src/EPR.Accreditation.Facade.Common/RESTservices/HttpSiteService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using EPR.Accreditation.Facade.Common.Dtos;
+using EPR.Accreditation.Facade.Common.Enums;
+using EPR.Accreditation.Facade.Common.RESTservices.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace EPR.Accreditation.Facade.Common.RESTservices
+{
+    public class HttpSiteService : BaseHttpService, IHttpSiteService
+    {
+        public HttpSiteService(
+            IHttpContextAccessor httpContextAccessor,
+            IHttpClientFactory httpClientFactory,
+            string baseUrl,
+            string endPointName) : base(httpContextAccessor, httpClientFactory, baseUrl, endPointName)
+        {
+        }
+
+        public async Task<Guid> CreateSite(Dtos.Site site)
+        {
```

Comment:
Your work doesn't have anything to do with creating a site, does it? If not, why are you doing this? If you need a site to dev test your work, you need to get one inserted into the db.

Reply:
Done.